text_recognizer/stems/paragraph.py

```diff
@@ -18,28 +18,51 @@ class ParagraphStem(ImageStem):
 
     scale_factor = IMAGE_SCALE_FACTOR
 
-    def __init__(self, augment=False, color_jitter_kwargs=None, random_affine_kwargs=None):
+    def __init__(
+        self,
+        augment=False,
+        color_jitter_kwargs=None,
+        random_affine_kwargs=None,
+        random_perspective_kwargs=None,
+        gaussian_blur_kwargs=None,
+        sharpness_kwargs=None,
+    ):
         super().__init__()
 
         if not augment:
             self.pil_transforms = transforms.Compose([transforms.CenterCrop(IMAGE_SHAPE)])
         else:
             if color_jitter_kwargs is None:
-                color_jitter_kwargs = {"brightness": (0.8, 1.6)}
+                color_jitter_kwargs = {"brightness": 0.4, "contrast": 0.4}
             if random_affine_kwargs is None:
                 random_affine_kwargs = {
-                    "degrees": 1,
-                    "shear": (-10, 10),
+                    "degrees": 3,
```

Comment:
I think we could turn these down ever so slightly -- we're clipping the first few characters on about 5-10% of examples.

see example below, which is entry 15 [here](https://wandb.ai/fullstackdeeplearning/fsdl-text-recognizer-2022-training/artifacts/run_table/run-2trhtbd4-trainpredictions/v98/files/train/predictions.table.json), see also entry 3

![image](https://user-images.githubusercontent.com/10442975/181310320-df48e9d4-7e09-4f44-bf24-8b08bcca6613.png)

Reply:
I stopped scaling up the images. It looks like it fixed the issue